src/services/artsData.js

```diff
@@ -1,5 +1,4 @@
-const baseUrl = process.env.REACT_APP_ARTS_DATA_URI;
-const publicUrl = process.env.REACT_APP_API_URL;
+const baseUrl = import.meta.env.VITE_APP_ARTS_DATA_URI;
```

Comment:
Forgot to add  const publicUrl = import.meta.env.VITE_APP_API_URL;
https://github.com/culturecreates/footlight-app/issues/1958#issuecomment-3242055290